packages/components-js/tests/e2e/puppeteer/specs/modal.e2e.ts

```diff
@@ -13,7 +13,6 @@ import {
   selectNode,
```

Comment:
in this file we import type `ModalAriaAttributes` which doesn't exist:

<img width="1099" alt="Bildschirm­foto 2023-03-10 um 11 59 57" src="https://user-images.githubusercontent.com/2981884/224299290-a9000765-e4b4-4fd6-9eef-26706af82b07.png">